### PR TITLE
Add recipe for duckpan.el

### PR DESCRIPTION
**Pull request by GuiltyDolphin**
Basic GitHub recipe.

Output from `check-recipe.el`:

screenshot from 2016-01-10 20 57 28

(I don't have any autoload cookies).

### CODE REVIEW

File: recipes/duckpan.rcp [hunk 1]
```diff
@@ -0,0 +1,8 @@
+(:name duckpan
+       :description "Tools for working with DuckDuckHack development"
+       :type github
+       :pkgname "duckpan-emacs"
+       :username "GuiltyDolphin"
```
Comment (npostavs) [inline recipes/duckpan.rcp]: `:username` is not supported since a couple of years I think, you should use `:pkgname "GuiltyDolphin/duckpan-emacs"` instead

**Comment by npostavs** [comment]
> (I don't have any autoload cookies).

Why don't you add some?

**Comment by GuiltyDolphin** [comment]
Added some cookies, fixed the username/pkgname fields.

New output from `check-recipe.el`:

screenshot from 2016-01-11 08 49 11

**Comment by GuiltyDolphin** [comment]
@dimitri 

Removed the non-used URL - should be all.

**Pull request merged**